packages/odata-service-inquirer/src/prompts/datasources/service-url/questions.ts

```diff
@@ -238,7 +238,7 @@ export function getServiceUrlQuestions({
         getIgnoreCertErrorsPrompt(connectValidator, requiredVersion)
     ];
 
-    if (getHostEnvironment() === hostEnvironment.cli) {
+    if (!PromptState.isYUI) {
```

Comment:
Can we revert to the existing use please?

Reply:
Reverted all changes